paparazzi/src/main/java/app/cash/paparazzi/PaparazziSdk.kt

```diff
@@ -302,7 +302,11 @@ public class PaparazziSdk @JvmOverloads constructor(
         }
       }
     } finally {
-      viewGroup.removeView(modifiedView)
+      // When using Dialogs in Compose, it seems that a WindowManager view gets added to the view group alongside the ComposeView under test.
+      // When our view under test is removed, compose will remove the WindowManager view first and then the ComposeView. But since the index of the view under test is 1
+      // and there are only 1 views in the ViewGroup when WindowManager is removed, it throws an invalid index.
```

Comment:
```suggestion
```

good context for a PR comment today.  But it's not a controversial code change (worst case, it's defensive) and the code comment will likely grow stale if Compose implementation details change.

Reply:
good point, I will move this to a PR comment for context. 

```

      // When using Dialogs in Compose, it seems that a WindowManager view gets added to the view group alongside the ComposeView under test.
      // When our view under test is removed, compose will remove the WindowManager view first and then the ComposeView. But since the index of the view under test is 1
      // and there are only 1 views in the ViewGroup when WindowManager is removed, it throws an invalid index.
      
```